atomicapp/nulecule/base.py

```diff
@@ -87,9 +88,20 @@ def unpack(cls, image, dest, config=None, namespace=GLOBAL_CONF,
             A Nulecule instance, or None in case of dry run.
         """
         logger.info('Unpacking image: %s to %s' % (image, dest))
-        docker_handler = DockerHandler(dryrun=dryrun)
-        docker_handler.pull(image)
-        docker_handler.extract(image, APP_ENT_PATH, dest, update)
+        if Utils.running_on_openshift():
+            _config = {
+                'providerapi': config['general']['providerapi'],
+                'accesstoken': config['general']['accesstoken'],
+                'namespace': config['general']['namespace']
+            }
+            op = OpenShiftProvider(_config, './', False)
```

Comment:
Add a comment to that effect. 

You could also simply just pass `config[GLOBAL_CONF]` directly (if you do, make sure to comment :))